apps/meteor/client/sidebarv2/RoomList/RoomList.tsx

```diff
@@ -51,11 +52,12 @@ const RoomList = () => {
 			<GroupedVirtuoso
 				groupCounts={groupsCount}
 				groupContent={(index) => (
-					<SidebarV2CollapseGroup
-						title={t(groupsList[index])}
-						onClick={() => handleClick(groupsList[index])}
-						onKeyDown={(e) => handleKeyDown(e, groupsList[index])}
-						expanded={!collapsedGroups.includes(groupsList[index])}
+					<RoomListCollapser
+						collapsedGroups={collapsedGroups}
+						handleClick={handleClick}
+						handleKeyDown={handleKeyDown}
```

Comment:
Could we keep the component event props as `onClick` and `onKeyDown`?
```suggestion
						onClick={handleClick}
						onKeyDown={handleKeyDown}
```

Reply:
The type is different from the default and I would have to omit from `HTMLAttributes`. Do you think it's really the best way? 
<img width="672" alt="image" src="https://github.com/user-attachments/assets/c3ad8fff-9cb5-44e3-b36f-6e067523ce24">

Reply:
I think omitting from `HTMLAttributes<HTMLElement>` makes sense regardless. If you leave as is the user can overwrite the event that calls `handleClick` and `handleKeyDown` which is not desired. 

I recommend doing the same you did with `onKeyDown` for `onClick` (e.g `onClick={(e) => onClick(e, groupTitle)}`) which will cover both the native and your custom event scenario.